shell/platform/android/platform_view_android.cc

```diff
@@ -65,21 +66,27 @@ void PlatformViewAndroid::SurfaceCreated(JNIEnv* env,
   }
 
   ANativeWindow_release(window);
+
+  auto gl_surface = std::make_unique<GPUSurfaceGL>(surface_gl_.get());
+  NotifyCreated(std::move(gl_surface), [this, backgroundColor] {
+    rasterizer().Clear(backgroundColor, GetSize());
```

Comment:
In my tests GetSize() gets the proper window dimensions for the EGL surface by the time PlatformViewAndroid::SurfaceCreated is called.

The screen should be fully cleared to the background color at SurfaceCreated time with no flash.  Additionally, this should eliminate the flash during rotation where we had been clearing the surface midway through the window rotate animation.